staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -0,0 +1,42 @@
+package metrics
+
+import (
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	etcdRequestLatencies = prometheus.NewHistogramVec(
```

Comment:
besides latency, we probably want to know the number of keys we update/read/delete.

for update, it can be inferred from the latency histogram since it is 1 to 1 mapping to update related operations. However, for read/delete we need separate metrics.

Reply:
sure